### PR TITLE
17.12

### PR DESCRIPTION
**Pull request by benfjohnson**
I verified that this company/address should return 2 nexuses in the response via http://fqa.avalara.net:8020/swagger/ui/index.html#/Companies

### CODE REVIEW

File: test/companies.spec.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ describe('Company Integration Tests', () => {
         expect(res).toBeDefined();
         expect(res.contacts.length).toEqual(1);
         expect(res.locations.length).toEqual(1);
-        expect(res.nexus.length).toEqual(3);
+        expect(res.nexus.length).toEqual(2);
```
Comment (benfjohnson) [inline test/companies.spec.js]: @ted-spence-avalara could you try pushing this change to your 17.12 PR to see if it resolves the build failure? I don't have push access, and the CI authentication is not working with my fork

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "avatax",
-  "version": "17.7.1",
+  "version": "17.12.0",
```
Comment (justinsoliz) [inline package.json]: we handle this with an npm tag, let's revert to 17.7.1. When the build passes in master i'll push up the new tag and bumped package.json

**Comment by benfjohnson** [comment]
@justinsoliz when do you anticipate these updates will be available in npm?

**Comment by justinsoliz** [comment]
@benfjohnson 
Ted and I unblocked the larger test issue on friday. Still working through a couple of failed tests but I would expect it to be up in the next day or so. 
https://travis-ci.org/avadev/AvaTax-REST-V2-JS-SDK/builds/325634929

**Comment by benfjohnson** [comment]
awesome! Thanks Justin.

**Pull request merged**